src/extensions/ignite/addModule.js

```diff
@@ -1,6 +1,16 @@
 const { trim } = require('ramda')
 
 module.exports = (plugin, command, context) => {
+  const getModuleName = (moduleName, options) => {
+    if (options.version) {
+      return `${moduleName}@${options.version}`
+    } else if (!options.yolo) {
+      context.print.warning(`Plugin should specify specific version for NPM module ${moduleName} in addModule call.`)
+      context.print.warning(`Please add version: "<version here>" or to suppress this warning, pass in yolo: true`)
+    }
+    return moduleName
```

Comment:
Okay, okay, I'll change it. 🙄 

Sidenote: I like multiple returns -- short circuit the rest of the function if it's not necessary, like a guard statement.

But I'm not dogmatic about it. I don't want your skin crawling. 😁